### PR TITLE
Create New Branch and Added Files

### PR DESCRIPTION
**Pull request by Madhava1104**

### CODE REVIEW

**Review by JayanthSL — APPROVED** [review]

**Review by JayanthSL — CHANGES_REQUESTED** [review]
Make sure Design as per requirement

File: lib/screens/contact_select.dart [hunk 1]
```diff
@@ -28,8 +30,8 @@ class ContactSelectScreen extends StatelessWidget {
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
           Container(
-            color: const Color(0xFF3f1e65),
-            padding: const EdgeInsets.symmetric(horizontal: 16.0),
+            color: Appearance.deepmulberry,
```
Comment (JayanthSL) [inline lib/screens/contact_select.dart:33]: Make in single line

File: lib/screens/contact_select.dart [hunk 2]
```diff
@@ -41,13 +43,13 @@ class ContactSelectScreen extends StatelessWidget {
                     fontWeight: FontWeight.bold,
```
Comment (JayanthSL) [inline lib/screens/contact_select.dart:43]: Use between w100-900

File: lib/screens/contact_select.dart [hunk 3]
```diff
@@ -41,13 +43,13 @@ class ContactSelectScreen extends StatelessWidget {
...
                   ),
                 ),
-                const SizedBox(height: 20),
+                const SizedBox(height: Dimensions.space20),
                 Container(
                   decoration: BoxDecoration(
-                    color: const Color(0xFF21182b),
-                    borderRadius: BorderRadius.circular(10),
+                    color: Appearance.darkeggplant,
```
Comment (JayanthSL) [inline lib/screens/contact_select.dart:49]: Single line

File: lib/screens/contact_select.dart [hunk 4]
```diff
@@ -59,28 +61,28 @@ class ContactSelectScreen extends StatelessWidget {
                             hintText: 'Search any contact / name',
                             hintStyle: TextStyle(color: Colors.white70),
                           ),
-                          style: TextStyle(color: Colors.white70), // Less opacity for text
+                          style: TextStyle(color: Colors.white70),
```
Comment (JayanthSL) [inline lib/screens/contact_select.dart:64]: Use appearance

File: lib/screens/contact_select.dart [hunk 5]
```diff
@@ -59,28 +61,28 @@ class ContactSelectScreen extends StatelessWidget {
...
                   ),
                 ),
-                const SizedBox(height: 20),
+                const SizedBox(height: Dimensions.space20),
               ],
             ),
           ),
           Container(
-            color: const Color(0xFF0e051a),
+            color: Appearance.ericblack,
```
Comment (JayanthSL) [inline lib/screens/contact_select.dart:75]: Use dimensions

File: lib/screens/contact_select.dart [hunk 6]
```diff
@@ -124,9 +126,9 @@ class ContactSelectScreen extends StatelessWidget {
         },
         icon: const Icon(Icons.add),
```
Comment (JayanthSL) [inline lib/screens/contact_select.dart:127]: Use string

File: lib/screens/home_screen.dart [hunk 7]
```diff
@@ -101,50 +103,50 @@ class _HomeContentState extends State<HomeContent> {
     final balanceFuture = context.findAncestorStateOfType<_HomeScreenState>()?._balanceFuture;
```
Comment (JayanthSL) [inline lib/screens/home_screen.dart:103]: Use question mark from string

File: lib/screens/home_screen.dart [hunk 8]
```diff
@@ -101,50 +103,50 @@ class _HomeContentState extends State<HomeContent> {
...
       child: CustomScrollView(
         slivers: [
           SliverToBoxAdapter(
             child: Stack(
               children: <Widget>[
                 Container(
-                  height: 250.0,
+                  height: Dimensions.space250,
                   width: double.infinity,
                   color: Color(getColorHexFromStr('#FDD148')),
```
Comment (JayanthSL) [inline lib/screens/home_screen.dart:115]: Use appearance

Comment (JayanthSL) [inline lib/screens/home_screen.dart:115]: Use dimensions

File: lib/screens/home_screen.dart [hunk 9]
```diff
@@ -101,50 +103,50 @@ class _HomeContentState extends State<HomeContent> {
...
+                  right: Dimensions.space100,
                   child: Container(
-                    height: 400.0,
-                    width: 400.0,
+                    height: Dimensions.space400,
+                    width: Dimensions.space400,
                     decoration: BoxDecoration(
-                      borderRadius: BorderRadius.circular(200.0),
+                      borderRadius: BorderRadius.circular(Dimensions.radius200),
                       color: Color(getColorHexFromStr('#FEE16D')).withOpacity(0.4),
```
Comment (JayanthSL) [inline lib/screens/home_screen.dart:125]: Use appearance n dimensions

File: lib/screens/home_screen.dart [hunk 10]
```diff
@@ -101,50 +103,50 @@ class _HomeContentState extends State<HomeContent> {
...
+                  left: Dimensions.space150,
                   child: Container(
-                    height: 300.0,
-                    width: 300.0,
+                    height: Dimensions.space300,
+                    width: Dimensions.space300,
                     decoration: BoxDecoration(
-                      borderRadius: BorderRadius.circular(150.0),
+                      borderRadius: BorderRadius.circular(Dimensions.radius150),
                       color: Color(getColorHexFromStr('#FEE16D')).withOpacity(0.5),
```
Comment (JayanthSL) [inline lib/screens/home_screen.dart:137]: Use dimensions n appearance

File: lib/screens/contact_select.dart (hunk 2 above)
Comment (Madhava1104) [reply lib/screens/contact_select.dart:43]: This how to do it???

File: lib/screens/contact_select.dart (hunk 1 above)
Comment (Madhava1104) [reply lib/screens/contact_select.dart:33]: Done

File: lib/screens/contact_select.dart (hunk 3 above)
Comment (Madhava1104) [reply lib/screens/contact_select.dart:49]: Done

File: lib/screens/contact_select.dart (hunk 4 above)
Comment (Madhava1104) [reply lib/screens/contact_select.dart:64]: Done

File: lib/screens/contact_select.dart (hunk 5 above)
Comment (Madhava1104) [reply lib/screens/contact_select.dart:75]: Done

File: lib/screens/home_screen.dart (hunk 10 above)
Comment (Madhava1104) [reply lib/screens/home_screen.dart:137]: Done

File: lib/screens/home_screen.dart (hunk 8 above)
Comment (Madhava1104) [reply lib/screens/home_screen.dart:115]: Double Infinity How to give in dimensions?

Comment (Madhava1104) [reply lib/screens/home_screen.dart:115]: Done

File: lib/screens/home_screen.dart (hunk 7 above)
Comment (Madhava1104) [reply lib/screens/home_screen.dart:103]: After paranthesis , question mark is there no that one?? (   ()?._balanceFuture  )

File: lib/screens/contact_select.dart (hunk 6 above)
Comment (Madhava1104) [reply lib/screens/contact_select.dart:127]: How??

File: lib/screens/contact_select.dart (hunk 2 above)
Comment (JayanthSL) [reply lib/screens/contact_select.dart:41]: Fontweight.w600

File: lib/widgets/appearance.dart [hunk 11]
```diff
@@ -82,6 +82,21 @@ class Appearance {
   static Color success700 = const Color(0xFF2F9461);
   static Color fogra39 = const Color(0xFF020202);
   static Color charcoal = const Color(0xFF363853);
+  static Color deepblue = const Color(0xFF290164);
```
Comment (JayanthSL) [inline lib/widgets/appearance.dart:85]: Remove all these don't change appearance.dart file

File: lib/screens/contact_select.dart [hunk 12]
```diff
@@ -10,48 +10,124 @@ class ContactSelectScreen extends StatelessWidget {
...
+              padding: const EdgeInsets.all(5.0),
+              decoration: BoxDecoration(
+                color: const Color(0xFF21182b),
+                borderRadius: BorderRadius.circular(16),
+              ),
+              margin: const EdgeInsets.symmetric(horizontal: 16.0),
+              child: ListView.separated(
+                itemCount: contacts.length,
+                separatorBuilder: (context, index) => const Divider(
+                    color: Color(0x80B0B0B0),
```
Comment (JayanthSL) [inline lib/screens/contact_select.dart]: Use dimensions

File: lib/screens/contact_select.dart [hunk 13]
```diff
@@ -10,48 +10,124 @@ class ContactSelectScreen extends StatelessWidget {
...
+            height: 20,
+          ),
+          Expanded(
+            child: Container(
+              padding: const EdgeInsets.all(5.0),
+              decoration: BoxDecoration(
+                color: const Color(0xFF21182b),
+                borderRadius: BorderRadius.circular(16),
+              ),
+              margin: const EdgeInsets.symmetric(horizontal: 16.0),
```
Comment (JayanthSL) [inline lib/screens/contact_select.dart]: Dimensions

File: lib/screens/sendmoney.dart [hunk 14]
```diff
@@ -42,148 +42,242 @@ class _SendMoneyScreenState extends State<SendMoneyScreen> {
...
   Widget build(BuildContext context) {
     return Scaffold(
-      backgroundColor: Colors.purple[900],
-      appBar: AppBar(
-        backgroundColor: Colors.purple[900],
-        elevation: 0,
-        leading: IconButton(
-          icon: const Icon(Icons.arrow_back, color: Colors.white),
-          onPressed: () => Navigator.of(context).pop(),
+      extendBodyBehindAppBar: true, // Make sure the scaffold extends behind the app bar
```
Comment (JayanthSL) [inline lib/screens/sendmoney.dart]: Remove comments bc chatgpt

**Review by JayanthSL — APPROVED** [review]

**Review by JayanthSL — APPROVED** [review]

**Review by Madhava1104 — COMMENTED** [review]
Reviewed

**Review by JayanthSL — APPROVED** [review]

**Review by JayanthSL — APPROVED** [review]

**Pull request closed without merging**